content/en/news/releases/1.12.x/announcing-1.12/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Istio 1.12
+linktitle: 1.12
+subtitle: Major Update
+description: Istio 1.12 release announcement.
+publishdate: 2021-11-16
+release: 1.12.0
+skip_list: true
+aliases:
+    - /news/announcing-1.12
+    - /news/announcing-1.12.0
+---
+
+We are pleased to announce the release of Istio 1.12!
+
+{{< relnote >}}
+
+This is the last release of 2021. We would like to thank the entire Istio community.
+
+{{< tip >}}
+Istio 1.12.0 is officially supported on Kubernetes versions `1.18.0` to `1.22.0`.
+{{< /tip >}}
+
+Here are some of the highlights of the release:
```

Comment:
I am working on a commit on this doc, if you plan to add to the blog feel free to message me to make sure we don't have duplicates

Reply:
We'll also need to update data/args.yaml with the supported versions.

Reply:
Also: istio.io/istio.io/content/en/docs/releases/supported-releases/index.md